### PR TITLE
Describe stata licensing

### PR DESCRIPTION
**Pull request by sebbacon**

### CODE REVIEW

File: docs/actions-scripts.md [hunk 1]
```diff
@@ -31,10 +31,35 @@ These Docker images have yet to be optimised; if you have skills in creating Doc
...
-automatically use the OpenSAFELY stata license. If not, get in touch if you
-need to apply your own license and we can help.
+As Stata is a commercial product, a license key is needed to use it. 
+
+#### If you are a member of the opensafely GitHub organisation
+* If you are using Windows, then the `opensafely` command line software will
+automatically use the OpenSAFELY stata license. 
+* If you are using macOS:
+   * Download and install the [github cli tool (`gh`)](https://cli.github.com/)
+   * Run `gh auth login --web`. Select the "HTTPS" option, and follow the instructions
```
Comment (StevenMaude) [inline docs/actions-scripts.md:41]: nit:
```suggestion
   1. Download and install [GitHub's command-line tool (`gh`)](https://cli.github.com/)
   2. Run `gh auth login --web`. Select the "HTTPS" option, and follow the instructions
```

File: docs/actions-scripts.md [hunk 2]
```diff
@@ -31,10 +31,35 @@ These Docker images have yet to be optimised; if you have skills in creating Doc
...
+  * The `opensafely` command line software will now automatically use the OpenSAFELY stata license
+
+#### All other external users
+
+If you are not a member of the opensafely GitHub organisation, you will need to provide your own Stata-MP license. Unfortunately other flavours of Stata are not yet supported; let us know if this is a problem.
+
+First, locate your Stata license string as follows:
```
Comment (StevenMaude) [inline docs/actions-scripts.md:48]: nit: I'd personally use an ordered list here for instructions for the reader to follow. Just to help emphasise where each step starts and end.

File: docs/actions-scripts.md [hunk 3]
```diff
@@ -31,10 +31,35 @@ These Docker images have yet to be optimised; if you have skills in creating Doc
...
-need to apply your own license and we can help.
+As Stata is a commercial product, a license key is needed to use it. 
+
+#### If you are a member of the opensafely GitHub organisation
+* If you are using Windows, then the `opensafely` command line software will
+automatically use the OpenSAFELY stata license. 
+* If you are using macOS:
+   * Download and install the [github cli tool (`gh`)](https://cli.github.com/)
+   * Run `gh auth login --web`. Select the "HTTPS" option, and follow the instructions
+  * The `opensafely` command line software will now automatically use the OpenSAFELY stata license
```
Comment (StevenMaude) [inline docs/actions-scripts.md:42]: nit:
```suggestion
  * The `opensafely` command line software will now automatically use the OpenSAFELY Stata license
```

File: docs/actions-scripts.md [hunk 4]
```diff
@@ -31,10 +31,35 @@ These Docker images have yet to be optimised; if you have skills in creating Doc
...
+* On Linux, somewhere like `/usr/local/stata17/`
+* On macOS it's usually in `/Applications/Stata/`
+
+Within that file, you'll find a license string of the format `SerialNumber!Code!Authorization!User!Organisation!VersionCode`. 
+
+Now set it as an environment variable, for example:
+
+    export STATA_LICENSE='your license string'
+
+The `opensafely` command line software will now automatically use this OpenSAFELY stata license.
```
Comment (StevenMaude) [inline docs/actions-scripts.md:62]: nit:
```suggestion
The `opensafely` command line software will now automatically use this Stata license.
```

File: docs/actions-scripts.md [hunk 5]
```diff
@@ -31,10 +31,35 @@ These Docker images have yet to be optimised; if you have skills in creating Doc
...
+* If you are using Windows, then the `opensafely` command line software will
+automatically use the OpenSAFELY stata license. 
+* If you are using macOS:
+   * Download and install the [github cli tool (`gh`)](https://cli.github.com/)
+   * Run `gh auth login --web`. Select the "HTTPS" option, and follow the instructions
+  * The `opensafely` command line software will now automatically use the OpenSAFELY stata license
+
+#### All other external users
+
+If you are not a member of the opensafely GitHub organisation, you will need to provide your own Stata-MP license. Unfortunately other flavours of Stata are not yet supported; let us know if this is a problem.
```
Comment (StevenMaude) [inline docs/actions-scripts.md:46]:
```suggestion
If you are not a member of the opensafely GitHub organisation, you must provide your own **Stata/MP** license. Unfortunately other Stata flavours are not yet supported; [let us know](how-to-get-help.md) if this is a problem.
```

File: docs/actions-scripts.md [hunk 6]
```diff
@@ -31,10 +31,35 @@ These Docker images have yet to be optimised; if you have skills in creating Doc
...
+
+First, locate your Stata license string as follows:
+
+Locate a text file, called STATA.LIC (on Windows) or stata.lic (macOS and Linux) which is usually at the top level of the folder of your Stata installation:
```
Comment (StevenMaude) [inline docs/actions-scripts.md:50]: nit:
```suggestion
Locate a text file, called `STATA.LIC` (on Windows) or `stata.lic` (macOS and Linux) which is usually at the top level of the folder of your Stata installation:
```

File: docs/actions-scripts.md (hunk 4 above)
Comment (bloodearnest) [reply docs/actions-scripts.md:62]: Also maybe add a line saying:

You will need to repeat this step each time you open a new terminal.

**Review by bloodearnest — APPROVED** [review]

**Pull request merged**